etc/systemd/dnf-system-upgrade.service

```diff
@@ -6,7 +6,7 @@ Documentation=http://www.freedesktop.org/wiki/Software/systemd/SystemUpdates
 DefaultDependencies=no
 Requires=sysinit.target
 After=sysinit.target systemd-journald.socket system-update-pre.target
-Before=shutdown.target system-update.target
+Before=poweroff.target shutdown.target system-update.target
```

Comment:
I am not sure if not to explicitly add also the `reboot.target` which is triggered by default with the `reboot` subcommand.

Reply:
I think it would make sense.